target/test-classes/BrowserProperties

```diff
@@ -0,0 +1 @@
+browser = local_chrome
```

Comment:
Everything which is in the target folder shouldn't be in a remote repository

Reply:
Fixed. Please, check.